lobby_server.yaml

```diff
@@ -14,7 +14,7 @@
 - version: "1.10"
   host: lobby.triplea-game.org
   port: 3304
-  http_server_uri: lobby.triplea-game.org:4567
+  http_server_uri: "http://lobby.triplea-game.org:4567"
```

Comment:
Should the schema be changed to indicate this is a `uri` instead of `host-name|ipv4|ipv6`?  I'm not even sure why the schema validator is allowing this.

I also just noticed that `http_server_uri` is not in the required properties list.  I'm not sure if that was an oversight or it was done on purpose to not exclude this property from the `1.9.0.0` entry.  If the addition of this property doesn't break 1.9 clients, I'd say just add it (a dummy value, if necessary) so we don't forget to include this property when we next add a new product version.

Reply:
Given the application is expecting a specific data type, it's reasonable to use the more constrained version.  5e0f40dbb changes the json type to URI

I'm not sure why I chose to use the prod URL, I updated that to be staging.

I'm not sure if we'll forget to add the property, more likely typo it maybe; the extra constraint is not a bad thing to have, I updated that in: e729044c5aa2bf45bc23147fb5c61e606a1db0d9